### DISCUSSION TITLE
UIGF v2.3 国际化兼容性建议

### DISCUSSION THREAD

**Discussion by Masterain98** [UIGF-org/.github#4] [General]
## Uniformed Interchangeable GachaLog Format standard of Json (UIGF.J)

`item_id` 必填，`Name`（物品名称）选填，以减少程序对不同语言记录文件的复杂判断

Name -> item_id 可以参考 https://api.uigf.org/dict/all.json

---
### 方案1
`item_type`在抽卡记录中表明了抽取物品的种类（武器或角色），是一个重要的字段。但是原神官方提供的接口中，该数据的值是受游戏客户端语言影响，会根据客户端语言返回对应翻译过的结果。祈愿导出工具如果不统一这一栏数据的结果，可能会导致祈愿导出/分析工具无法判断物品的种类。
考虑到目前仅有两种物品种类，建议将角色在`item_type`中记录为字串符`"1"`，将武器在`item_type`中记录为`"2"`

> 在获取抽卡记录 URL 时，抽卡导出工具可以统一设置`lang`参数为指定语言，来简化程序对数据的处理，比如 `lang=en`

### 方案2
删除 `item_type`一栏

---

## Uniformed Interchangeable GachaLog Format standard of Workbook (UIGF.W)
在`原始数据`表中跟随全部的 UIGF.J v2.3 的更新

**Comment by Lightczx** [comment]
https://github.com/UIGF-org/UIGF-org.github.io/pull/11
@Masterain98

**Reply by Masterain98** [reply]
> [UIGF-org/UIGF-org.github.io#11](https://github.com/UIGF-org/UIGF-org.github.io/pull/11) @Masterain98

tmr
